modules/arm_plugin/src/transformations/convert_interpolate_arm.cpp

```diff
@@ -109,7 +124,6 @@ ArmPlugin::pass::ConvertInterpolate::ConvertInterpolate() {
         auto arm_interp = std::make_shared<opset::ArmInterpolate>(interp->input_value(0),
                                                                   interp->input_value(1),
                                                                   interp->input_value(2),
-                                                                  interp->input_value(3),
```

Comment:
Looks like input_value(1) and input_value(2) could be removed as well. Neither of them are used inside ArmInterpolate conversion

Reply:
Could it be reduced to `arm_interp = std::make_shared<opset::ArmInterpolate>(interp->input_value(0), attrs);`?